src/main/java/com/fleotadezuta/youthprogrammanager/facade/ChildParentFacade.java

```diff
@@ -27,6 +30,7 @@ public class ChildParentFacade {
     private final ParentMapper parentMapper;
     private final ParentService parentService;
     private final ChildService childService;
+    private final TicketTypeService ticketTypeService;
```

Comment:
Úgy látom ez végül nem lett használva. Ha így van, akkor ki is törölheted a property-k közül.